src/cozmo/objects.py

```diff
@@ -283,6 +284,8 @@ def __init__(self, *a, **kw):
         self.last_tapped_time = None
 
         #: int: The robot's timestamp of the last tapped event
+        #: ``None`` if the cube wasn't tapped yet
+        #: In milliseconds relative to robot epoch
```

Comment:
Needs punctuation - Sphinx will run these lines together and it doesn't look good without it (plus i've been trying to be good about punctuation for Sphinx docs in general)